packages/amplication-server/src/env.ts

```diff
@@ -29,6 +29,10 @@ export class Env {
   static readonly AUTH_ISSUER_CLIENT_SECRET = "AUTH_ISSUER_CLIENT_SECRET";
   static readonly AUTH_ISSUER_CALLBACK_URL = "AUTH_ISSUER_CALLBACK_URL";
 
+  static readonly AUTH0_DOMAIN = "AUTH0_DOMAIN";
+  static readonly AUTH0_CLIENT_ID = "AUTH0_CLIENT_ID";
+  static readonly AUTH0_CLIENT_SECRET = "AUTH0_CLIENT_SECRET";
```

Comment:
Are these needed or can we just use the current env vars:
- `AUTH_ISSUER_CLIENT_ID`
- `AUTH_ISSUER_CLIENT_SECRET`

Reply:
I'm not sure they are the same